.github/workflows/pull_request_comments.yml

```diff
@@ -0,0 +1,49 @@
+name: Pull Request Comments
+
+on:
+  workflow_run:
+    workflows: [Pull Request]
+    types:
+      - completed
+
+jobs:
+  download:
+    runs-on: ubuntu-latest
+    if: >
+      github.event.workflow_run.event == 'pull_request' &&
+      github.event.workflow_run.conclusion == 'success'
```

Comment:
Did this run in the PRB? Would we expect it to, or is that a sign that something is up?